db/migrate/20200323161500_backfill_sent_to_provider_at.rb

```diff
@@ -0,0 +1,21 @@
+class BackfillSentToProviderAt < ActiveRecord::Migration[6.0]
+  def up
+    ApplicationChoice.all.each do |application_choice|
+      state_change = application_choice.audits.find do |audit|
+        audit.audited_changes['status'] == %w[application_complete awaiting_provider_decision]
+      end
+
+      next unless state_change
+
+      application_choice.update!(
+        sent_to_provider_at: state_change.created_at,
```

Comment:
Is `state_change.created_at` not the same as `40.business_days.before(application_choice.reject_by_default_at)`? We wouldn't need to use the audit log if this was true.

Reply:
No, I think where this breaks is for applications in `awaiting_provider_decision`... right?

Reply:
I think not because the `reject_by_default_at` is always at the end of the day, so it's not entirely accurate.

Reply:
True